oscm-ui-tests/src/test/java/org/oscm/marketplace/MarketplacePlaygroundAccountWT.java

```diff
@@ -24,6 +25,9 @@
 import org.oscm.webtest.MarketplacePathSegments;
 import org.oscm.webtest.PortalTester;
 
+// FIXME remove ignore and fix
```

Comment:
After adding the correct constant in MarketplacePathSegments.java, the Ignore tag is not necessary